ReleaseProcess.md

```diff
@@ -6,7 +6,6 @@ The following is a step-by-step procedure for releasing a new version of Google-
 1. Ensure all changelists are code-reviewed and have +1
 1. `cd gson` to the parent directory; ensure there are no open files and all changes are committed.
 1. Run `mvn release:clean`
-1. Do a dry run: `mvn release:prepare -DdryRun=true`
```

Comment:
Have removed this because it looks like this would prevent subsequent (non-dry-run) `mvn release:prepare` from working properly?